CHANGELOG.md

```diff
@@ -19,3 +24,4 @@ For detailed information on the tasks carried out during this release, please se
 
 [v7.0.0.1]: https://github.com/wso2/docker-ei/compare/v6.5.0.3...v7.0.0.1
 [v7.0.0.2]: https://github.com/wso2/docker-ei/compare/v7.0.0.1..v7.0.0.2
+[v7.0.0.3]: https://github.com/wso2/docker-ei/compare/v7.0.0.2..v7.0.0.3
```

Comment:
@RidmiR since we have to release Micro Integrator as well, we need to perform the JDK upgrade to MI profile as well. Images need to be consistent.